app/controllers/concerns/medium_response.rb

```diff
@@ -1,7 +1,7 @@
 module MediumResponse
   def medium_response(medium)
     scope = Medium.where(id: medium)
-    headers['ETag'] = gen_etag(scope)
+    headers['ETag'] = gen_etag(scope.to_a)
```

Comment:
Check we don't use this on index routes with the ability to convert large scopes to arrays

Reply:
this is safe as the `medium_response` is only used for export resources - i've renamed this module / method for clairty